drivers/doc.txt

```diff
@@ -43,3 +43,10 @@
  * @ingroup     drivers
  * @brief       Drivers for storage devices
  */
+
+/**
+ * @defgroup    drivers_can CAN Drivers
+ * @ingroup     drivers
+ * @ingroup     can
```

Comment:
> renaming it to something like "communication devices"

Do you mean renaming "Network Device Drivers" (drivers_netdev) to "Communication Device Drivers" (drivers_communication) ?